sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1346,6 +1346,96 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
+  test("writing partitioned DataFrame to Hive compatible format") {
+    val schema = StructType(
+      Array(StructField("a", LongType, false),
+        StructField("b", StringType, false),
+        StructField("c", StringType, false),
+        StructField("d", IntegerType, false)))
+
+    val rdd = sparkContext.parallelize(Array(
+      Row(1L, "test", "NW", 1),
+      Row(2L, "test2", "SF", 1),
+      Row(3L, "test3", "NW", 1),
+      Row(1L, "test", "CA", 2)))
+
+    val df = sqlContext.createDataFrame(rdd, schema)
+    df.write.partitionBy("c", "d").saveAsTable("testDFTABLE1")
```

Comment:
We can't create a table without dropping it at the end of test. How about creating a method that creates the table and use it in next test?